src/cpp/session/modules/SessionConnectionsInstaller.R

```diff
@@ -388,7 +388,26 @@
    writeLines(lines, odbcinstPath)
 })
 
-.rs.addFunction("odbcBundleRegisterLinux", function(name, driverPath, version) {
+.rs.addFunction("odbcOdbcInstLibPath", function() {
+   odbcinstLib <- NULL
+
+   odbcinstBin <- system2("which", "odbcinst", stdout = TRUE)
+   if (is.null(odbcinstBin) || length(odbcinstBin) == 0) {
+      warning("Could not find path to odbcinst.")
+   }
+   else {
+      odbcinstLink <- system2("readlink", odbcinstBin, stdout = T)
+      if (!is.null(odbcinstLink) && length(odbcinstLink) > 0) {
```

Comment:
As above (is `length` intended?)

Reply:
Yes, intended. Until, Kevin suggested using `Sys,readLink()` which requires an `NA` check.